ILSANG/Sources/Views/Setting/DeleteAccountView.swift

```diff
@@ -36,14 +36,18 @@ struct DeleteAccountView: View {
                     
                     Text("일상 탈퇴 전 확인하세요!")
                         .foregroundColor(.black)
+                        .font(.system(size: 23))
+                        .fontWeight(.bold)
                         .padding(.bottom, 11)
 
                     Text("탈퇴하시면 모든 데이터는 복구가 불가능합니다.")
+                        .font(.system(size: 15))
                         .foregroundColor(Color.gray400)
                         .padding(.bottom, 35)
 
                     
                     Text("• 진행 및 완료된 모든 퀘스트 내용이 삭제됩니다.\n• 사장님이 관리하는 단골 데이터에서 삭제됩니다.")
```

Comment:
```suggestion
                    Text("• 진행 및 완료된 모든 퀘스트 내용이 삭제됩니다.\n• 경험치 및 레벨은 복구가 불가합니다.")
```
이 부분만 이 PR에서 같이 수정해주시면 좋을 것 같습니다!

Reply:
적용했습니다.
피그마에서는 디자인이 변경되지 않았는데, 상단에 "탈퇴하시면 모든 데이터는 복구가 불가능합니다."가 중복되는 것이 맞나요??

Reply:
<img width="280" alt="image" src="https://github.com/TeamFair/ILSANG-iOS/assets/100195563/67954f2c-f2a4-4d15-8d80-9004f90b0c97">
@HIBAR97 피그마에도 문구는 변경되어있습니다! 
중복되는 문구가 수정이 필요해보이시면 신디한테 물어보면 어떨까요?

Reply:
수정되었군요?
오늘 질문해 보도록 하겠습니다.